### PR TITLE
feat: 282 switch tok04 osa21 to per DC

### PR DESCRIPTION
**Pull request by stafaniasaju**
### Description

- Enabled PER for DC location tok04 and osa21
- Added validation for prefix

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [x] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

**Review by surajsbharadwaj — CHANGES_REQUESTED** [review]
Change IBM provider version to 1.63.0 for all DA solutions

File: modules/pi-sap-system-type1/main.tf [hunk 1]
```diff
@@ -112,14 +114,15 @@ module "pi_hana_storage_calculation" {
...
-  version = "1.0.3"
+  version = "1.1.0"
 
   pi_workspace_guid          = var.pi_workspace_guid
   pi_instance_name           = local.pi_hana_instance_name
   pi_ssh_public_key_name     = var.pi_ssh_public_key_name
   pi_image_id                = var.pi_hana_instance.image_id
   pi_networks                = local.pi_networks
   pi_sap_profile_id          = var.pi_hana_instance.sap_profile_id
+  pi_boot_image_storage_tier = "tier3"
```
Comment (surajsbharadwaj) [inline modules/pi-sap-system-type1/main.tf:125]: This should be tier 1

**Review by surajsbharadwaj — CHANGES_REQUESTED** [review]
Also e2e must be updated to use 1.1.0 version

**Comment by stafaniasaju** [comment]
> Also e2e must be updated to use 1.1.0 version

e2e source refers to module which is already updated.
source     = "../../modules/pi-sap-system-type1" 

If this is not the intended part, can you point me to the line?

https://github.com/terraform-ibm-modules/terraform-ibm-powervs-sap/blob/a67954758b3711cced1e0aa7981c125378c734b9/modules/pi-sap-system-type1/main.tf#L48

File: modules/pi-sap-system-type1/main.tf (hunk 1 above)
Comment (surajsbharadwaj) [reply modules/pi-sap-system-type1/main.tf:125]: This has to be fixed @stafaniasaju

**Pull request closed without merging**